setup-environment-internal

```diff
@@ -360,5 +364,19 @@ Your build environemnt has been configured with:
     SDKMACHINE = ${SDKMACHINE}
     DISTRO = ${DISTRO}
 
-You can now run 'bitbake ledge-iot'
+Build for a specific MACHINE:
+	bitbake ledge-iot
+	bitbake ledge-gateway
+Build on multiconfig:
+	- set the config to use on auto.conf with BBMULTICONFIG variable
+	  export BBMULTICONFIG="qemuarm stm32mp157c-dk2 ti-am572x"
+	  sed -e "s%^BBMULTICONFIG ?= .*$%BBMULTICONFIG ?= \"${MULTICONFIG}\"%" -i conf/auto.conf
+	- Build with multiconfig
+	  MACHINE=ledge-qemuarm bitbake \
+		mc:qemuarm:ledge-gateway \
+		mc:qemuarm:ledge-iot \
+		mc:stm32mp157c-dk2:u-boot-ledge mc:stm32mp157c-dk2:optee-os mc:stm32mp157c-dk2:arm-trusted-firmware-ledge \
+		mc:ti-am572x:u-boot-ledge mc:ti-am572x:optee-os
+
+For futher insformation read conf/multiconfig/README.txt
 EOF
```

Comment:
How do the cnahges affectthe WIC image generation?
What wic images will you get when compiling with multiconfig?

Reply:
it's doesn't change the wic generation, via the command of multiconfig you reqest to generate only the imlages for qemu machine and not for other and for other machine just generate the mandatory firmware for boot.